clang/test/CodeGenSYCL/kernel-caller-entry-point.cpp

```diff
@@ -10,9 +10,9 @@
 // RUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple amdgcn-amd-amdhsa -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-AMDGCN %s
 // RUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple nvptx-nvidia-cuda -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-NVPTX %s
 // RUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple nvptx64-nvidia-cuda -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-NVPTX %s
-// RUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple spir-unknown-unknown -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-SPIR %s
+// INTEL XUN: %clang_cc1 -fsycl-is-device -emit-llvm -aux-triple x86_64-pc-windows-msvc -triple spir-unknown-unknown -std=c++17 %s -o - | FileCheck --check-prefixes=CHECK-DEVICE,CHECK-SPIR %s
```

Comment:
Why only these two runs are being disabled?

Reply:
Excellent question! These two run configurations result in a failed "Unsupported host architecture" assertion. The really interesting question is why similar failures don't occur for the corresponding run configurations that target 64-bit Linux or for the 32-bit nvptx device target with either 64-bit host. I brought this up in the Intel SYCL upstream meeting yesterday and the conclusion was "bugs". I'll submit a github issue to follow up.

Hmm, the "INTEL" annotation isn't appropriate for this repo. I'll go ahead and remove these run lines instead.